src/app/filter-column/filter-column.model.ts

```diff
@@ -189,12 +189,12 @@ export class Filter {
   public static STATUS_OUT = new Filter( ParticipantColumn.STATUS_OUT, Filter.TEXT_TYPE );
   public static STATUS_IN = new Filter( ParticipantColumn.STATUS_IN, Filter.TEXT_TYPE );
   public static CARE_EVOLVE = new Filter( ParticipantColumn.CARE_EVOLVE, Filter.CHECKBOX_TYPE );
-  public static RESULT_TEST = new Filter(ParticipantColumn.RESULT_TEST, Filter.JSON_ARRAY_TYPE, null, new NameValue( ParticipantColumn.RESULT_TEST.name, null ),
-  false, true, null, null, null, null, false, false, false, false, Filter.TEXT_TYPE );
-  public static CORRECTED_TEST = new Filter(ParticipantColumn.CORRECTED_TEST, Filter.JSON_ARRAY_TYPE, null, new NameValue( ParticipantColumn.CORRECTED_TEST.name, null ),
-  false, true, null, null, null, null, false, false, false, false, Filter.CHECKBOX_TYPE );
-  public static TIME_TEST = new Filter(ParticipantColumn.TIME_TEST, Filter.JSON_ARRAY_TYPE, null, new NameValue( ParticipantColumn.TIME_TEST.name, null ),
-  false, true, null, null, null, null, false, false, false, false, Filter.DATE_TYPE );
+  public static RESULT_TEST = new Filter( ParticipantColumn.RESULT_TEST, Filter.JSON_ARRAY_TYPE, null, new NameValue( ParticipantColumn.RESULT_TEST.name, "'" ),
```

Comment:
"'" ?

Reply:
single quote, needs to be there because of the fact that in the json some elements have. single quote and some like date are numbers and don't